ssl/s3_lib.c

```diff
@@ -2360,8 +2360,7 @@ static SSL_CIPHER ssl3_ciphers[] =
      SSL_NOT_DEFAULT | SSL_HIGH,
      SSL_HANDSHAKE_MAC_SHA384 | TLS1_PRF_SHA384,
      256,
-     256
-    },
+     256},
```

Comment:
Oh, that apparently depends on whether there's a comma after the last element. I fixed those but missed one. Will undo.